cmd/deploy/deploy.go

```diff
@@ -92,26 +91,24 @@ func Deploy(ctx context.Context) *cobra.Command {
 			// This is needed because the deploy command needs the original kubeconfig configuration even in the execution within another
 			// deploy command. If not, we could be proxying a proxy and we would be applying the incorrect deployed-by label
 			os.Setenv(model.OktetoWithinDeployCommandContextEnvVar, "false")
-			if okteto.IsOkteto() && options.Namespace != "" {
-				create, err := utils.ShouldCreateNamespace(ctx, options.Namespace)
+
+			if err := contextCMD.LoadManifestV2WithContext(ctx, options.Namespace, options.ManifestPath); err != nil {
```

Comment:
Having to rename this import because it collides with a built-in package is a smell that indicating about some of the previous comments made on package organization.